Mathlib/Data/Int/GCD.lean

```diff
@@ -9,10 +9,10 @@ Authors: Sangwoo Jo (aka Jason), Guy Leroy, Johannes Hölzl, Mario Carneiro
 ! if you have ported upstream changes.
 -/
 import Mathlib.Data.Nat.GCD.Basic
+import Mathlib.Algebra.GroupPower.Lemmas
```

Comment:
was this intentional?

Reply:
Yes, `Mathlib.Tactic.NormNum` used to import this, and it's the source of [`Int.natAbs_pow` and `isUnit_ofPowEqOne`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/GroupPower/Lemmas.html#Int.natAbs_pow)